src/main/java/com/dev/cinema/model/Ticket.java

```diff
@@ -0,0 +1,53 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
```

Comment:
```suggestion
@Table(name = "ticket")
```
let's use either singular or plural for all the tables